CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 ### Fixed
+- Width of the dropdown lists becomes equal with the selector box by default, rather than expanding as wide as the screen width regardless of the selector box width (#20).
+- Setting a custom `width` style for `listStyle` now works as intended (#20).
+- Setting `center` for a custom `alignSelf` style for `listStyle` will now center it correctly (#20).
```

Comment:
We should probably make the `#20` link to the actual PR

Reply:
Also, I think it's common practice to leave an empty line between a markdown header and any content that follows it

Reply:
Fixed.